### ISSUE TITLE
Adding a class using the class attributes appends the class without separation.

### ISSUE THREAD

**Issue by esiao** [pug-php/pug#62]
Simple issue fixable on user side but seems like a missing feature / bug to me.

Taking this code

```
- $classes = 'MyClasss'

.MyInitialClass(class=$classes)
```

It returns

```
<div class="MyInitialClassMyClass"></div>
```

Expected behaviour

```
<div class="MyInitialClass MyClass"></div>
```

Adding a space in my class fixes it but here the example is simplified, I use arrays most of the time and need to add a space to the first entry.

**Comment by kylekatarnls** [comment]
Thanks for report, not noticed this regression, this will be my top priority.

**Comment by kylekatarnls** [comment]
Merged in 2.4.1
